src/tile/tile.cpp

```diff
@@ -433,7 +434,8 @@ static void writeOutputFile(const std::string& filename, pdal::PointViewPtr view
 void addArgs(pdal::ProgramArgs& programArgs, BaseInfo::Options& options, pdal::Arg * &tempArg, pdal::Arg * &threadsArg)
 {
     programArgs.add("output,o", "Output directory/filename", options.outputDir);
-    programArgs.add("files,i", "Input files/directory", options.inputFiles).setPositional();
+    programArgs.add("files,i", "Input files/directory", options.inputFiles);
```

Comment:
is the removal of setPositional() necessary? e.g. in `vpc.cpp` it is still there, and I guess it would avoid the strange looking syntax `--files=data1.las --files=data2.las`

Reply:
Fixed by using `parseSimple()` as in other tools. This call does not perform validation of arguments, only checks validity of the argument values.